README.md

```diff
@@ -16,15 +16,15 @@ A gem that works with _Florence_ to deterministically calculate whether an **act
 
 For Deliveroo Employees:
 
-- Many people contribute to Determinator and Florence. We hang out in [this Slack channel](slack://channel?team=T03EUNC3F&id=C7437816J)
+- Many people contribute to Determinator and Florence. We hang out in [this Slack channel](https://deliveroo.slack.com/app_redirect?channel=florence_wg)
 - [This JIRA board](https://deliveroo.atlassian.net/secure/RapidBoard.jspa?rapidView=156) covers pieces of work that are planned or in-flight
 - [This Workplace group](https://deliveroo.facebook.com/groups/1893254264328414/) holds more general discussions about the Florence ecosystem
 
 At the moment we can only promise support for Determinator within Deliveroo, but if you add [issues to this github repo](https://github.com/deliveroo/determinator/issues) we'll try and help if we can!
 
-## Usage
+## Basic Use
 
-Once [set up](#installation), determinator can be used to determine whether a **feature flag** or **experiment** is on or off for the current user and, for experiments, which **variant** they should see.
+Once [set up](#installation), determinator can be used to determine whether a **feature flag** or **experiment** is on or off for the current actor (or user) and, for experiments, which **variant** they should see.
```

Comment:
I think the initial `feature_flag_on?` example could do with fleshing out. My first time reading this, I didn't grok that we must have either a guid or an actor set.

Reply:
I tried to cover that with the Installation section and the further usage, but you're right, just referencing it isn't enough :P I'll add an example!